pkg/controller/backup_controller.go

```diff
@@ -82,7 +84,8 @@ func NewBackupController(
 		client:       client,
 		queue:        workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "backup"),
 
-		clock: &clock.RealClock{},
+		clock:  &clock.RealClock{},
+		logger: logrus.New(),
```

Comment:
I'm not very familiar with the setup/configuration of logrus.  Would it be more appropriate to pass this into the controllers as an argument instead of having the controllers bootstrap their own individually? 

Specifically if we were to want to add in the ability to use structured logging or any of the other hooks it supports, would this be easy to accomplish?

Reply:
I agree, we'll want to set up logging once, as high as we can, and pass it down.